src/screens/pos-actions/process-transactions/process-transactions.tsx

```diff
@@ -387,6 +387,97 @@ export class ProcessTransactionsComponent extends React.Component<
         this.props.closeProcessTransactions();
     }
 
+    @bind
+    private onPressSignTransaction(transaction: IBlockchainTransaction) {
+        this.props.signAndSendTransactions([transaction]);
+    }
+
+    @bind
+    private signAllTransactions() {
+        // sign all transactions - HD wallet
+        this.props.signAndSendTransactions(this.props.transactions);
+    }
+
+    public renderBottomButton() {
+        const { styles } = this.props;
+        if (this.props.transactions.length === 0) return <View />;
+
+        let allTransactionPublished = 0;
+        let transactionsFailed = 0;
+
+        for (const tx of this.props.transactions) {
+            if (this.isTransactionPublished(tx)) {
+                allTransactionPublished++;
+            }
+            if (tx.status === TransactionStatus.DROPPED || tx.status === TransactionStatus.FAILED) {
+                transactionsFailed++;
+            }
+        }
+
+        if (this.props.selectedWallet.type === WalletType.HD) {
+            // is HD wallet
+
+            if (allTransactionPublished === this.props.transactions.length) {
+                const disabledButton =
+                    transactionsFailed !== 0
+                        ? false
+                        : !(allTransactionPublished === this.props.transactions.length);
+
+                return (
+                    <Button
+                        primary
+                        onPress={this.onPressContinue}
+                        wrapperStyle={styles.continueButton}
+                        disabled={disabledButton}
+                    >
+                        {translate('App.labels.continue')}
+                    </Button>
+                );
+            } else
+                return (
+                    <Button
+                        primary
+                        onPress={this.signAllTransactions}
+                        wrapperStyle={styles.continueButton}
+                        disabled={false}
+                    >
+                        {translate('Transaction.signAll')}
+                    </Button>
+                );
+        } else {
+            // is Ledger wallet
+            if (this.state.currentIndex < this.props.transactions.length) {
+                return (
+                    <Button
+                        primary
+                        onPress={() =>
+                            this.onPressSignTransaction(
+                                this.props.transactions[this.state.currentIndex]
+                            )
+                        }
+                        wrapperStyle={styles.continueButton}
+                        disabled={false}
+                    >
+                        {`${translate('App.labels.sign')} ${formatPlural(
```

Comment:
this approach is wrong, Not all languages have the same order of words, for example `Sign 1st transaction` in german is `Erste Transaktion unterschreiben`. 
PS: Erste = 1st :)